conformance-suites/1.0.3/conformance/resources/webgl-test-utils.js

```diff
@@ -2641,7 +2636,11 @@ var startPlayingAndWaitForVideo = function(video, callback) {
 
   video.loop = true;
   video.muted = true;
-  video.play();
+  try {
+    await video.play();
+  } catch (error) {
+    testFailed('Video failed to play(): ' + error);
+  }
```

Comment:
Let's not backport this, since it's a little risky, and we're just doing it to try to make things better for ourselves on tip

Reply:
Sorry I tested the first patch, but the not the second one. Yes it needs async function. I'll use async on tip and then restore to my previous if (video.error) check on earlier versions.